frontend/coreproject/src/lib/components/pages/anime_info/AnimeMainDetails.svelte

```diff
@@ -0,0 +1,198 @@
+<script lang="ts">
+	export let anime: any;
+
+	import { truncate_str } from '$lib/helpers/truncate_str';
+	// icons
+	import PlayCircle from '$icons/PlayCircle.svelte';
+	import Book from '$icons/Book.svelte';
+	import HeadPhones from '$icons/HeadPhones.svelte';
+	import Video from '$icons/Video.svelte';
+	import Edit3 from '$icons/Edit3.svelte';
```

Comment:
What is this ?

Reply:
It was the icons I need to add

Reply:
> It was the icons I need to add
 
 
 Please dont add arbitrary values. Use the edit type in `Edit.svelte`